osu.Game/Graphics/UserInterface/OsuAnimatedButton.cs

```diff
@@ -73,17 +76,25 @@ public OsuAnimatedButton()
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load()
         {
             if (AutoSizeAxes != Axes.None)
             {
                 content.RelativeSizeAxes = (Axes.Both & ~AutoSizeAxes);
                 content.AutoSizeAxes = AutoSizeAxes;
             }
+        }
 
-            Enabled.BindValueChanged(enabled => this.FadeColour(enabled.NewValue ? Color4.White : colours.Gray9, 200, Easing.OutQuint), true);
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Colour = enablementColour;
+            Enabled.BindValueChanged(_ => this.FadeColour(enablementColour, 200, Easing.OutQuint));
         }
 
+        private Color4 enablementColour => Enabled.Value ? Color4.White : colours.Gray9;
```

Comment:
The term "enablement" really throws me off. Saw it somewhere else recently and had to double-take.

Reply:
Does "base" make more sense? Any suggestions?

Reply:
`dimColour` maybe? i dunno, i'm fine with how you have it, just makes me double take. i'd prefer `enabledColour` but that probably makes less sense.

Reply:
Renamed to `dimColour`